language/documentation/book/src/modules-and-scripts.md

```diff
@@ -25,10 +25,8 @@ arguments, and must not return a value. Here is an example with each of these co
 
```

Comment:
I think this whole file needs some works. Probably need to introduce the `module <addr>::<name>` syntax. 
And talk about how named addresses are opaque for resolving modules, in other words you need to use the name always, and cant use the value passed in by the command line
Similarly maybe also talk about  one module per address, regardless of name.

Reply:
I agree, but the changes here look like a strict improvement over what we currently have. Can we go ahead with these for now and come back for more later?